README.md

```diff
@@ -12,5 +12,20 @@
 
 ## Usage
 
+1. Put your config file in `conf/config.json`
```

Comment:
> b/c the CMD is set to conf.json by default

I see that now; thanks.

> I've added a check to script/deploy

👍  

> I assumed we could fall back on synapse's failure to start there, but I can (and did) add a check in the script itself.

Three cheers for failing early and clearly.